transports/quic/src/lib.rs

```diff
@@ -52,7 +52,7 @@
 //! `Endpoint` manages a background task that processes all incoming packets.  Each
 //! `QuicConnection` also manages a background task, which handles socket output and timer polling.
 
-#![forbid(unstable_features, unsafe_code)]
+#![forbid(unstable_features, unsafe_code, intra_doc_link_resolution_failure)]
```

Comment:
```suggestion
#![forbid(unstable_features, unsafe_code)]
```

I removed all of these on purpose, as it's checked automatically by the CI.